R/runWithComparison.R

```diff
@@ -22,10 +23,17 @@
 #' @importFrom utils head
 #' @export
 runWithComparison <- function(renvInstallPackages,
-                              computations = baseComputations[c("magpiePrep", "remindPrep")],
+                              computations = c("magpiePrep", "remindPrep"),
                               piktestsFolder = getwd(),
                               diffTool = c("delta", "colordiff", "diff"), ...) {
   stopifnot(!is.null(renvInstallPackages))
+  if (!is.list(computations)) {
```

Comment:
This should be the same as in piktests::run:
```suggestion
  if (is.character(computations)) {
```
and so on